cargo-apk/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 0.6.0 (2021-05-06)
+
+- Added support for android string resource id in application label.
```

Comment:
That's an implicit result of not using the application name as apk name anymore? Perhaps mention that more explicitly within the next bullet point instead.